docs/ad/api.md

```diff
@@ -1108,60 +1443,153 @@ POST _opendistro/_alerting/monitors
 
 #### Request
 
 ```json
 GET _opendistro/_anomaly_detection/detectors/<detectorId>/_profile/
 GET _opendistro/_anomaly_detection/detectors/<detectorId>/_profile?_all=true
 GET _opendistro/_anomaly_detection/detectors/<detectorId>/_profile/<type>
+GET /_opendistro/_anomaly_detection/detectors/<detectorId>/_profile/<type1>,<type2>
 ```
+
 #### Sample Responses
 
 ```json
-GET _opendistro/_anomaly_detection/detectors/4j1313EBhPlEUyl3nsX-/_profile
+GET _opendistro/_anomaly_detection/detectors/<detectorId>/_profile
 
 {
     "state":"DISABLED",
     "error":"Stopped detector: AD models memory usage exceeds our limit."
 }
 
-GET _opendistro/_anomaly_detection/detectors/m4ccEnIBTXsGi3mvMt9p/_profile?_all=true&pretty
+GET _opendistro/_anomaly_detection/detectors/<detectorId>/_profile?_all=true&pretty
 
 {
-  "state" : "RUNNING",
-  "models" : [
+  "state": "RUNNING",
+  "models": [
     {
-      "model_id" : "cneh7HEBHPICjJIdXdrR_model_rcf_2",
-      "model_size_in_bytes" : 4456448,
-      "node_id" : "VS29z70PSzOdHiEw4SoV9Q"
+      "model_id": "cneh7HEBHPICjJIdXdrR_model_rcf_2",
+      "model_size_in_bytes": 4456448,
+      "node_id": "VS29z70PSzOdHiEw4SoV9Q"
     },
     {
-      "model_id" : "cneh7HEBHPICjJIdXdrR_model_rcf_1",
-      "model_size_in_bytes" : 4456448,
-      "node_id" : "VS29z70PSzOdHiEw4SoV9Q"
+      "model_id": "cneh7HEBHPICjJIdXdrR_model_rcf_1",
+      "model_size_in_bytes": 4456448,
+      "node_id": "VS29z70PSzOdHiEw4SoV9Q"
     },
     {
-      "model_id" : "cneh7HEBHPICjJIdXdrR_model_threshold",
-      "node_id" : "Og23iUroTdKrkwS-y89zLw"
+      "model_id": "cneh7HEBHPICjJIdXdrR_model_threshold",
+      "node_id": "Og23iUroTdKrkwS-y89zLw"
     },
     {
-      "model_id" : "cneh7HEBHPICjJIdXdrR_model_rcf_0",
-      "model_size_in_bytes" : 4456448,
-      "node_id" : "Og23iUroTdKrkwS-y89zLw"
+      "model_id": "cneh7HEBHPICjJIdXdrR_model_rcf_0",
+      "model_size_in_bytes": 4456448,
+      "node_id": "Og23iUroTdKrkwS-y89zLw"
     }
   ],
-  "shingle_size" : 8,
-  "coordinating_node" : "Og23iUroTdKrkwS-y89zLw",
-  "total_size_in_bytes" : 13369344
+  "shingle_size": 8,
+  "coordinating_node": "Og23iUroTdKrkwS-y89zLw",
+  "total_size_in_bytes": 13369344,
+  "init_progress": {
+    "percentage": "70%",
+    "estimated_minutes_left": 77,
+    "needed_shingles": 77
+  }
 }
 
-GET _opendistro/_anomaly_detection/detectors/m4ccEnIBTXsGi3mvMt9p/_profile/total_size_in_bytes
+GET _opendistro/_anomaly_detection/detectors/<detectorId>/_profile/total_size_in_bytes
 
 {
   "total_size_in_bytes" : 13369344
 }
 ```
 
+If you have configured the category field, you can see the number of unique values in the field and also all the active entities with models running in memory.
+You can use this data to estimate the memory required for anomaly detection to help decide the size of your cluster.
+For example, if a detector has one million entities and only 10 of them are active in memory, then you need to scale up your cluster.
```

Comment:
Can we change to "scale up or out your cluster"?  Scale up is upgrading to a more powerful instance types, while scale out is adding more nodes.